### PR TITLE
chore(release): Hook beta testing

### PR DESCRIPTION
**Pull request by kddejong**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by kddejong** [comment]
@ezbeard need to figure out the error code for a hook failure.  I think its exit code 254 but need to suppress it and compare it.

File: resources/beta-buildspec-python.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ phases:
       - pip install -r requirements-dev.txt
       - cfn --version
       - sam --version
-
+      - sam build --use-container --build-image public.ecr.aws/sam/build-python3.7
```
Comment (ericzbeard) [inline resources/beta-buildspec-python.yml:20]: Is `cfn submit` going to keep doing the build with the hard coded image anyway?

File: hooks/beta-buildspec-python.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  build:
+    commands:
+      - echo Entered the build phase...
+      - cfn submit --set-default
+      - INTEG_SUCCEED_STACK_NAME="integ-$(echo $ENTRY_PATH | sed s/_/-/g)"
+      - INTEG_FAIL_STACK_NAME="integ-$(echo $ENTRY_PATH | sed s/_/-/g)"
+      - aws cloudformation create-stack --stack-name $INTEG_SUCCEED_STACK_NAME --template-body file://test/integ-succeed.yml
+      - aws cloudformation wait stack-create-complete --stack-name $INTEG_SUCCEED_STACK_NAME
+      - aws cloudformation create-stack --stack-name $INTEG_FAIL_STACK_NAME --template-body file://test/integ-fail.yml
+      - aws cloudformation wait stack-create-complete --stack-name $INTEG_FAIL_STACK_NAME
```
Comment (ericzbeard) [inline hooks/beta-buildspec-python.yml:29]: We need to catch this error or the job will fail when it's supposed to succeed (and succeed when it's supposed to fail)

File: hooks/S3_BucketVersioningEnabled/test/integ-succeed.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+Resources:
+  Bucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      VersioningConfiguration:
```
Comment (ericzbeard) [inline hooks/S3_BucketVersioningEnabled/test/integ-succeed.yml:5]: I wonder if we're going to end up needing to adopt the convention from the `inputs` folder where there are multiple tests, for example `integ-succeed-0.yml`. For now this is Ok but we should consider it.

File: resources/beta-buildspec-python.yml (hunk 1 above)
Comment (kddejong) [reply resources/beta-buildspec-python.yml]: Didn't mean to include that was testing something.

File: hooks/S3_BucketVersioningEnabled/test/integ-succeed.yml (hunk 3 above)
Comment (kddejong) [reply hooks/S3_BucketVersioningEnabled/test/integ-succeed.yml:5]: I think we will

**Review by ericzbeard — APPROVED** [review]

**Pull request merged**